packages/binding-http/src/http-server.ts

```diff
@@ -187,7 +187,8 @@ export default class HttpServer implements ProtocolServer {
       // fill in binding data
       for (let address of Helpers.getAddresses()) {
         for (let type of ContentSerdes.get().getOfferedMediaTypes()) {
-          let base: string = this.scheme + "://" + address + ":" + this.getPort() + "/" + encodeURIComponent(title);
+          let baseURI: string = thing.base || ( this.scheme + "://" + address + ":" + this.getPort() );
```

Comment:
What happens if both are present?

Reply:
> What happens if both are present?
What do you mean by both?  it either uses the base attribute (as a URL prefix), or generates based on the network interface address it is using in the for loop.  The network interface address is always present.  It is only overridden if base attribute is set.
Am I missing your question?

> What happens with different bindings (CoAP, HTTP, ..)
Good question.  This is my first exposure to CoAP, I'll look into that.

Reply:
Thanks!